lib/facebook/messenger/incoming/message_reaction.rb

```diff
@@ -0,0 +1,31 @@
+module Facebook
+  module Messenger
+    module Incoming
+      # The Message reaction class represents an incoming Facebook Messenger message reaction
+      # @see https://developers.facebook.com/docs/messenger-platform/reference/webhook-events/message-reactions
+      class MessageReaction
+        include Facebook::Messenger::Incoming::Common
+
+        def payload
+          @messaging['reaction']
+        end
+
+        def reaction
+          @messaging['reaction']['reaction']
+        end
+
+        def emoji
+          @messaging['reaction']['reaction']
```

Comment:
I think this should be `@messaging['reaction']['emoji']`:
![image](https://user-images.githubusercontent.com/2923637/71983210-90efc480-3226-11ea-8625-572abfff8fb4.png)

Reply:
Yeah you are right :)